addons/im_livechat/tests/test_get_discuss_channel.py

```diff
@@ -59,7 +59,7 @@ def test_channel_get_livechat_visitor_info(self):
         )
         self.assertEqual(
             data["res.partner"],
-            [
+            self._filter_partners_fields(
```

Comment:
Note: changing `_filter_persona_fields` was not necessary diff but since I added `_filter_messages_fields` with a better API I took the opportunity to update it too.